content/agents/overview.md

```diff
@@ -28,77 +28,102 @@ Cloudify includes a set of agent packages with support for:
   winrm set winrm/config/service/auth @{Basic="true"}
   netsh advfirewall firewall add rule name="WinRM 5985" protocol=TCP dir=in localport=5985 action=allow
-  {{< /gsHighlight >}}
-
-
+  ```
 {{% gsNote title="Note" %}}
 1.  The commands above are provided in a syntax that is suitable for invocation from a command-prompt window. If you
     are using userdata (or an equivalent feature), you might need to adjust the commands to accommodate its requirements(for example:
     if these commands are to be run within a batch file, each line must be prefixed with `call`).
 2.  The commands are very permisive and must adjusted according to your requirements. These settings provide
-    unencrypted WinRM access to the machine. 
+    unencrypted WinRM access to the machine.
     From MSDN: `AllowUnencrypted` - Enables the client computer to request unencrypted traffic.
 {{% /gsNote %}}
 
-## Pre-requisites for Init Script and Provided Agent Installations
 
-To use the `init_script` and `provided` installation methods, you must use an IaaS plugin that supports it. Currently, the Openstack and AWS plugins support these installation methods.
+* `init_script` - For systems that don't have SSH/WinRM access, userdata
+(e.g. cloud-init) may be used. In this method, an install script is
+rendered, and a temporary link pointing to it is created. This link is
+then embedded in a separate `download script`, which is injected into
+the agent host's userdata. When the host is booted for the first time,
+the download script is executed. It downloads the installation script
+from the temporary link and executes it.
+To use the `init_script` method, the Iaas provider and Cloudify plugin
+need to support userdata. Currently, the Openstack
+and AWS plugins support this installation method.
+
+* `plugin` - For systems that have neither SSH/WinRM access, nor
+the ability to inject a script into userdata, the `plugin` installation
+method allows implementing a custom way to download and execute the
+installation script. In this method, a temporary download link for the
+installation script is created and made available to the plugin using
+the `ctx`. The plugin can then implement a custom method for downloading
+and executing the installation script on the agent's host.
+
+* `provided` - For systems that require the agent to be embedded in the
+image, users can use the `provided` method. In this mode, it is up to
+the user to make sure the agent is already installed on the image.
+During the `install` workflow, a configuration script will be rendered
+and a temporary link to it will created and made available
+via Cloudify's logs. This script is
+similar to the installation script, except that it doesn't download or
+install the agent package, but only configures and starts the agent
+daemon. This script needs to be downloaded and executed manually for
+every agent.
+
+* `none` - In some cases, you cannot or prefer not to install an agent
```

Comment:
it seems this is the only paragraph that uses the personal "you", everything else says "the user", do we want to make it all be the same?